### ISSUE TITLE
[Bug Report] v-chip close lacks accessibility label

### ISSUE THREAD

**Issue by gabaum10** [vuetifyjs/vuetify#11165]
### Environment
**Vuetify Version:** 2.2.22
**Vue Version:** 2.6.11
**Browsers:** Chrome 81.0.4044.113
**OS:** Linux x86_64

### Steps to reproduce
Load a `v-chip` component with "close" enabled.  The close button fails accessibility tests because it have a label.

![noDiscernableText](https://user-images.githubusercontent.com/1561245/79755148-9dd08980-82e6-11ea-8076-4532c4d0b8d9.png)

Additionally, how come this is not a slot?  I could fix it myself with a slot if that were an option.

### Expected Behavior
To either have a slot I could pass this in, or a way to specify a label for the close button

### Actual Behavior
Not able to do that.

### Reproduction Link
<a href="https://codepen.io/gabaum10/pen/bGVwqyY" target="_blank">https://codepen.io/gabaum10/pen/bGVwqyY</a>

**Comment by jyburns** [comment]
Hi, @KaelWD I would like to work on a resolution for this issue.
